src/MediaAtom/YoutubeOverlay.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { css } from 'emotion';
+import { Pillar } from '@guardian/types/Format';
+
+import { MediaMeta } from './MediaMeta';
+
+const overlayStyles = (image: string) => css`
+    background-image: url(${image});
+    background-size: cover;
+    background-position: 49% 49%;
+    background-repeat: no-repeat;
+    z-index: 0;
+    text-align: center;
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    max-height: 100vh;
+    cursor: pointer;
+`;
+
+const BottomLeft = ({
+    children,
+}: {
+    children: JSX.Element | JSX.Element[];
+}) => (
+    <div
+        className={css`
+            position: absolute;
+            bottom: 12px;
```

Comment:
Do these magic numbers relate to any particular thing that you can assign to a var/ comment to give them context?

Reply:
Thanks, I've used `space` here now from Source.